lib/utils/validate-addons.spec.js

```diff
@@ -0,0 +1,13 @@
+/* eslint node/no-unsupported-features: 0 */
+'use strict';
+
+describe('validate-addons', () => {
+	//eslint-disable-next-line
+	const {validateAddons} = require('initialize.mock');
+
+	it('Should validate multiple packages if supplied', async () => {
+		let itValidatesAddon = await validateAddons(['webpack-addons-ylvis', 'webpack-addons-noop']);
+		// BUG: We are making the values strigs, so the tests pass
```

Comment:
Me neither, but remove the .toString() and it fails :( I think it's that we use async/await.